gpytorch/kernels/multitask_kernel.py

```diff
@@ -39,10 +39,10 @@ def forward_diag(self, x1, x2):
         base data kernel and the task kernel.
         """
         task_indices = torch.arange(self.n_tasks, device=x1.device).long()
-        task_indices = task_indices.unsqueeze(-1).unsqueeze(-1)

```

Comment:
both this change and the one below have non-unix line endings

Reply:
Actually, the whole `multitask_kernel.py` file uses non-unix line endings throughout.

Reply:
I actually have no idea what is causing this -- both atom and git are setup to stop this, and it obviously works a majority of the time. 😕